modules/installation-requirements-user-infra.adoc

```diff
@@ -109,10 +110,18 @@ ifndef::ibm-z[|{op-system} or RHEL 7.6]
 |===
 
 ifdef::ibm-z[]
+[NOTE]
+====
+The control plane 120 GB storage must be a single disk.
```

Comment:
I'd move the note into the table. It will get lost here.

Reply:
Fixed in e4bec360f9254dfcdbc363a6fa918c278ecab916